coordinator/tests/resources/test-data/fhir/dispensing/Process-Request-Dispense-Not-To-Be-Despensed-200_OK.json

```diff
@@ -0,0 +1,1129 @@
+{
```

Comment:
Typo in the file name. Also, why shouldn't this be dispensed? It might be useful to try find a name that makes the issue with the request easily identifiable, e.g. `Process-Request-Dispense-400_Invalid_Checksum`. This will allow people to easily reuse the same payload for other use cases (unless this is a really specific one), and will make it easier to understand why this request cannot be dispensed.

Reply:
good point Its a dispense Notification with statusReasonCodeableConcept